### DISCUSSION TITLE
Couldn't get pass auth

### DISCUSSION THREAD

**Discussion by mwei0210** [xelaj/mtproto#62] [Q&A]
Was following the phone auth but stuck at AuthSendCode, i think i has all the input correct but getting these errors:

`&errors.withStack{
  error: &errors.withMessage{
    cause: &errors.withStack{
      error: &errors.withMessage{
        cause: &errors.errorString{
          s: "context canceled",
        },
        msg: "reading length",
      },
      stack: &errors.stack{
        0x5e4096,
        0x5e51a5,
        0x46a6e1,
      },
    },
    msg: "reading from connection",
  },
  stack: &errors.stack{
    0x5e51f4,
    0x46a6e1,
  },
}
&errors.withStack{
  error: &errors.withMessage{
    cause: &errors.withStack{
      error: &errors.withMessage{
        cause: &mtproto.ErrResponseCode{
          Code:           4294966892,
          Message:        "",
          Description:    "",
          AdditionalInfo: nil,
        },
        msg: "Server response error",
      },
      stack: &errors.stack{
        0x5e33d6,
        0x5e528d,
        0x46a6e1,
      },
    },
    msg: "decoding received data",
  },
  stack: &errors.stack{
    0x5e54f8,
    0x46a6e1,
  },
}`

**Comment by quenbyako** [comment]
@mwei0210 sorry for long response, could you please paste here binary answer from telegram server (hexdump-formated would be amazing)

**Reply by mwei0210** [reply]
No worries, where do i extract that from?
